.github/workflows/Build-and-deploy-reporting-services.yaml

```diff
@@ -0,0 +1,188 @@
+name: Build and push data-reporting-service image to ECR
+on:
+  push:
+    branches:
+      - main
+      - master
+      - rel-**
+#       Uncomment the following line only to test the build deploy from private branches.
+      - CNDIT-*
```

Comment:
Make sure to comment this out when you merge to main

Reply:
Done